README.md

```diff
@@ -24,4 +24,22 @@ make setup
 
 ```sh
 yarn run start:dev
+```
+
+
+## Run integration tests locally
+
+This can be handy when debugging the submission service and avoid rebuilds
+
+Start the services
+```sh
```

Comment:
Why aren't we instructing them to run `make test_integration`?

Reply:
added another section for running integration in docker.

sometimes its handy to be able to run integration tests with local build (yarn start:dev)